go.mod

```diff
@@ -47,23 +47,31 @@ require (
 	k8s.io/utils v0.0.0-20230726121419-3b25d923346b
 )
 
+require github.com/openshift-online/ocm-common v0.0.0
+
+replace github.com/openshift-online/ocm-common v0.0.0 => github.com/xueli181114/ocm-common v0.0.0-20240424104851-f2d7d7bc1b19
```

Comment:
@yasun1 Here
`Please notice this line is just temporary import. It will be removed after https://github.com/openshift-online/ocm-common/pull/45 is merged.
replace github.com/openshift-online/ocm-common v0.0.0 => github.com/xueli181114/ocm-common v0.0.0-20240424104851-f2d7d7bc1b19`